<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bullsncows</class>
 <widget class="QMainWindow" name="Bullsncows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Bullsncows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="gameField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>740</width>
      <height>650</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>260</width>
      <height>125</height>
     </rect>
    </property>
    <property name="text">
     <string>Начните игру</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>165</y>
      <width>220</width>
      <height>125</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите число:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>220</width>
      <height>125</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showRecords">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>220</width>
      <height>125</height>
     </rect>
    </property>
    <property name="text">
     <string>Рекорды</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkNumber">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>165</y>
      <width>220</width>
      <height>125</height>
     </rect>
    </property>
    <property name="text">
     <string>Проверить!</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLine">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>165</y>
      <width>260</width>
      <height>125</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
